core/src/main/scala/org/apache/spark/status/storeTypes.scala

```diff
@@ -97,6 +97,41 @@ private[spark] class StageDataWrapper(
   private def completionTime: Long = info.completionTime.map(_.getTime).getOrElse(-1L)
 }
 
+/**
+ * This object map the indices names of successful tasks' metrices. Mapped to short strings
+ * to save space when using a disk store.
+ */
+private[spark] object SuccessTaskIndexNames {
```

Comment:
Was this necessary to make new set of identifiers here -- why is it different?

Reply:
This new set of identifers is for mapping the index name of success table.If I use TaskIndexNames for mapping, it will results all the tasks instead of success tasks